docs/pages/guides/notifications-services.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Notifications
+description: Learn about different push notification services.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+
+Expo apps can work with any notification service or capabilities offered by the operating system thanks to [config plugins](/config-plugins/introduction/). The following services provide purpose-built Expo integrations for implementing push notifications in your app:
+
+> The [`expo-notifications`](/versions/latest/sdk/notifications/) library is designed and tested to work with Expo's push notification service. Some advanced features may not be compatible with third-party providers, as they often have their own native and React Native SDKs optimized for their services.
+
+### Expo Notifications
```

Comment:
Let's update these H3s to H2s and H4s to H3s in this guide. Generally, having semantic components following their order helps with SEO a lot.